validation.md

```diff
@@ -5,4 +5,4 @@ title: Device Support Validation
 permalink: /validation
 ---
 
-The device support validation documentation has moved to [{{site.docurl}}]({{https://developer.android.com/agi/start#device-validation}}).
```

Comment:
The removed link here looks correct for the validation page, but it's not used in the webpage output(https://gpuinspector.dev/validation). So, not sure if we should actually change the 'site.docurl' part here.

Reply:
site.docurl resolves to `https://developer.android.com/agi` which would be the text the user actually sees. The hyperlink is wrapped in `{{ }}` and my guess is that it's trying to resolve that but couldn't find a variable named  `https://developer.android.com/agi/start#device-validation` making it link to itself.

Reply:
So, I think we should change the text to show 'site.validation-url' as well. Currently it will still show docurl but point to somewhere else, right? 
Interestingly all other pages on https://gpuinspector.dev/docs/ just says 'The documentation has moved' without giving direct links (e.g. https://gpuinspector.dev/docs/devices is not forwarding to https://developer.android.com/agi/supported-devices).

Reply:
done